playwright/src/main/java/com/microsoft/playwright/impl/Connection.java

```diff
@@ -177,11 +180,8 @@ private JsonArray currentStackTrace() {
     return result;
   }
 
-  public ChannelOwner waitForObjectWithKnownName(String guid) {
-    while (!objects.containsKey(guid)) {
-      processOneMessage();
-    }
-    return objects.get(guid);
+  public PlaywrightImpl initializePlaywright() {
```

Comment:
It's a layering violation when the connection knows about playwright impl types but I think it's fine for now.